contracts/signatures/NonceVerifiable.sol

```diff
@@ -19,9 +19,16 @@ abstract contract NonceVerifiable is OwnableUpgradeable {
     /// @custom:schema (contract address -> token id -> signer address -> nonce)
     mapping(address => mapping(uint256 => mapping(address => uint256))) public assetNonce;
 
-    event ContractNonceUpdated(uint256 _from, uint256 _to, address _sender);
-    event SignerNonceUpdated(uint256 _from, uint256 _to, address _signer, address _sender);
-    event AssetNonceUpdated(uint256 _from, uint256 _to, address _contractAddress, uint256 _tokenId, address _signer, address _sender);
+    event ContractNonceUpdated(uint256 _from, uint256 _to, address indexed _sender);
```

Comment:
I don't feel comfortable using `indexed` for not the first parameters. Can we change the order and also the name of the parameters?

`_from` -> `_oldNonce`
_to` -> `_newNonce`